ggml.c

```diff
@@ -3386,6 +3412,34 @@ inline static void ggml_vec_gelu_f32(const int n, float * y, const float * x) {
 }
 #endif
 
+inline static float ggml_gelu_quick_f32(float x) {
+    return x*(1.0f/(1.0f+expf(GELU_QUICK_COEF*x)));
```

Comment:
Is there a reason why this isn't just `x / (1.0f + expf(GELU_QUICK_COEF * x))`?

Reply:
The only reason I can think of is that there is a [reciprocal instruction](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_rcp_ss) that should make 1/x faster than other divisions. However, [gcc doesn't seem to use](https://godbolt.org/z/dfo7dfhEP) in this code.

Reply:
I guess it's just following the python impl:

https://github.com/huggingface/transformers/blob/ee88ae59940fd4b2c8fc119373143d7a1175c651/src/transformers/activations.py#L94-L97

But it's irrelevant since we end up using the lookup table anyway

Reply:
I know it doesn't matter, but anyway here is a test: https://quick-bench.com/q/tmJMSzU789MADWT67QSLGed7AqQ
`rcp_ss` is slower, @JohannesGaessler version is ~10% faster